docs/source/installation.rst

```diff
@@ -88,13 +88,14 @@ Development version install
 
 Development versions of ``sktime`` are installs from static snapshots of the ``sktime`` repository.
```

Comment:
how about we rename this to "install of latest version", and the third setup we call consistently "developer setup"?

Avoiding the term "developer" or "development". Then we can say, git install of latest is not the developer setup or similar.